config/lib/express.js

```diff
@@ -279,6 +279,7 @@ module.exports.initHelmetHeaders = function (app) {
         '*.twitter.com',
         '*.google-analytics.com',
         '*.gstatic.com', // Google analytics related
+        '*.addthis.com', // addthis.com social media sharing service
```

Comment:
Looks like they use also `*.addthisedge.com`. You can see CSP warnings in console log by keeping it open when opening the page.